### ISSUE TITLE
Strange issue with H5177 as battery hits zero

### ISSUE THREAD

**Issue by wcbonner** [wcbonner/GoveeBTTempLogger#79]
This is going to require more investigation. I'm hoping that the fact that I remove known devices from BlueZ hourly means that the daily graph will stop being updated in the next hour. I don't understand why I'm getting data that appears valid from BlueZ and it's value is obviously wrong. I may want to add valid temperature ranges to the decode routines for each model.

![gvh-A4C1380D3B10-day](https://github.com/user-attachments/assets/77e2f604-547c-4603-a576-1ab6e8bc8107)
![gvh-A4C1380D3B10-year](https://github.com/user-attachments/assets/59a00ed5-5b97-4507-b325-b57374491f3f)

```
wim@WimPi4-Sola:~ $ tail --lines 200 /var/log/goveebttemplogger/gvh-A4C1380D3B10-2024-10.txt
2024-10-22 19:44:14     -52.4287        28.7    0
2024-10-22 19:44:17     -52.4287        28.7    0
2024-10-22 19:44:33     -52.4287        28.7    0
2024-10-22 19:44:36     13.1581 58.1    0
2024-10-22 19:44:43     13.1581 58.1    0
2024-10-22 19:44:47     13.1581 58.1    0
2024-10-22 19:44:50     13.1581 58.1    0
2024-10-22 19:45:01     -52.4287        28.7    0
2024-10-22 19:45:04     -52.4287        28.7    0
2024-10-22 19:45:10     -52.4287        28.7    0
2024-10-22 19:45:14     -52.4287        28.7    0
2024-10-22 19:45:17     -52.4287        28.7    0
2024-10-22 19:45:21     -52.4287        28.7    0
2024-10-22 19:45:29     -52.4287        28.7    0
2024-10-22 19:45:36     -52.4287        28.7    0
2024-10-22 19:45:43     -52.4287        28.7    0
2024-10-22 19:45:46     -52.4287        28.7    0
2024-10-22 19:45:49     -52.4287        28.7    0
2024-10-22 19:45:53     -52.4287        28.7    0
2024-10-22 19:45:57     -52.4287        28.7    0
2024-10-22 19:46:01     -52.4287        28.7    0
2024-10-22 19:46:05     -52.4287        28.7    0
2024-10-22 19:46:10     -52.4287        28.7    0
2024-10-22 19:46:14     -52.4287        28.7    0
2024-10-22 19:46:19     -52.4287        28.7    0
2024-10-22 19:46:23     -52.4287        28.7    0
2024-10-22 19:46:28     -52.4287        28.7    0
2024-10-22 19:46:34     -52.4287        28.7    0
2024-10-22 19:46:38     -52.4287        28.7    0
2024-10-22 19:46:42     -52.4287        28.7    0
2024-10-22 19:46:48     -52.4287        28.7    0
2024-10-22 19:46:52     -52.4287        28.7    0
2024-10-22 19:46:59     13.2581 58.1    0
2024-10-22 19:47:02     13.2581 58.1    0
2024-10-22 19:47:05     13.2581 58.1    0
2024-10-22 19:47:09     13.2581 58.1    0
2024-10-22 19:47:13     13.2582 58.2    0
2024-10-22 19:47:17     13.2582 58.2    0
2024-10-22 19:47:23     13.2582 58.2    0
2024-10-22 19:47:27     13.2582 58.2    0
2024-10-22 19:47:31     13.2582 58.2    0
2024-10-22 19:47:35     13.2582 58.2    0
2024-10-22 19:47:38     13.2582 58.2    0
2024-10-22 19:47:49     -52.4287        28.7    0
2024-10-22 19:47:53     -52.4287        28.7    0
```

**Comment by wcbonner** [comment]
It should have been removed here:
```
wim@WimPi4-Sola:~ $ journalctl -r -u goveebttemplogger.service
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-E38EC8C1989A-day.svg With Title: Sola H5074 (E3:8E:C8:C1:98:9A)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-E3605923147D-day.svg With Title: Sola Refrigerator H5074 (E3:60:59:23:14:7D)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-E36059218065-day.svg With Title: Sola Icebox H5074 (E3:60:59:21:80:65)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-E35ECC215C0F-day.svg With Title: Outside H5074 (E3:5E:CC:21:5C:0F)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-D03533334403-day.svg With Title: H5105 (D0:35:33:33:44:03)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-C33635306177-day.svg With Title: H5104 (C3:36:35:30:61:77)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-C23533302550-day.svg With Title: H5100 (C2:35:33:30:25:50)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C138DCCC3D-day.svg With Title: H5174 (A4:C1:38:DC:CC:3D)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C138D5A33B-day.svg With Title: Sola V-Berth H5074 (A4:C1:38:D5:A3:3B)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C13837BCAE-day.svg With Title: Sola Display H5075 (A4:C1:38:37:BC:AE)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 13:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C13805C7A1-day.svg With Title: Sola Aft Cabin H5074 (A4:C1:38:05:C7:A1)
Oct 22 13:09:44 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-E36059218065-cache.txt
Oct 22 13:08:43 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-A4C138D5A33B-cache.txt
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: StartDiscovery
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_37_BC_AE
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_E3_8E_C8_C1_98_9A
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_C3_36_35_30_61_77
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_E3_5E_CC_21_5C_0F
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_D5_A3_3B
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_E3_60_59_23_14_7D
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_E3_60_59_21_80_65
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_DC_CC_3D
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_05_C7_A1
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_C2_35_33_30_25_50
Oct 22 13:08:12 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_D0_35_33_33_44_03
Oct 22 13:08:11 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: StopDiscovery
Oct 22 13:07:42 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-E38EC8C1989A-cache.txt
Oct 22 13:07:42 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-E35ECC215C0F-cache.txt
```
and it's definitely got new data logged after 20:08
```
2024-10-22 20:07:47     -52.4287        28.7    0
2024-10-22 20:07:51     -52.4287        28.7    0
2024-10-22 20:07:54     -52.4287        28.7    0
2024-10-22 20:07:58     -52.4287        28.7    0
2024-10-22 20:08:02     -52.4287        28.7    0
2024-10-22 20:08:17     -52.4287        28.7    0
2024-10-22 20:08:24     -52.4287        28.7    0
2024-10-22 20:08:28     -52.4287        28.7    0
2024-10-22 20:08:32     -52.4287        28.7    0
2024-10-22 20:08:36     -52.4287        28.7    0
2024-10-22 20:08:48     -52.4287        28.7    0
2024-10-22 20:08:55     -52.4287        28.7    0
2024-10-22 20:09:00     -52.4287        28.7    0
2024-10-22 20:09:04     -52.4287        28.7    0
2024-10-22 20:09:09     -52.4287        28.7    0
2024-10-22 20:09:13     -52.4287        28.7    0
2024-10-22 20:09:17     -52.4287        28.7    0
2024-10-22 20:09:21     -52.4287        28.7    0
2024-10-22 20:09:38     -52.4287        28.7    0
2024-10-22 20:09:42     -52.4287        28.7    0
2024-10-22 20:09:46     -52.4287        28.7    0
2024-10-22 20:09:56     -52.4287        28.7    0
2024-10-22 20:10:00     -52.4287        28.7    0
2024-10-22 20:10:05     -52.4287        28.7    0
2024-10-22 20:10:09     -52.4287        28.7    0
2024-10-22 20:10:16     -52.4287        28.7    0
2024-10-22 20:10:19     -52.4287        28.7    0
2024-10-22 20:10:22     -52.4287        28.7    0
2024-10-22 20:10:25     -52.4287        28.7    0
2024-10-22 20:10:32     -52.4287        28.7    0
2024-10-22 20:10:36     -52.4287        28.7    0
2024-10-22 20:10:39     -52.4287        28.7    0
2024-10-22 20:10:44     -52.4287        28.7    0
```

**Comment by wcbonner** [comment]
I'd removed the battery from the H5177 device around 9:30pm. the display was completely off prior, but the bluetooth may have been responding.

The last time I'm removing the device from BlueZ known devices is at 9:38pm, but I'm still updating the daily graph and cache for nearly two hours after that.

for some reason I was still recording data in the data log with the out of range values. the data log is UTC values. I'm 7 hours off UTC.

```
wim@WimPi4-Sola:~ $ journalctl -b 0 -r -u goveebttemplogger.service | grep A4_C1_38_0D_3B_10
Oct 22 21:38:29 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 21:08:28 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 20:38:27 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 20:08:26 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 19:38:25 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
Oct 22 19:08:24 WimPi4-Sola goveebttemplogger[926]: /org/bluez/hci0: org.bluez.Adapter1: RemoveDevice /org/bluez/hci0/dev_A4_C1_38_0D_3B_10
```

```
wim@WimPi4-Sola:~ $ journalctl -b 0 -r -u goveebttemplogger.service | grep A4C1380D3B10
Oct 23 00:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 23 00:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-year.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 23 00:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-month.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 23 00:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 23 00:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 23 00:00:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:55:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:50:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:45:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:40:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:30:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:25:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:23:48 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-A4C1380D3B10-cache.txt
Oct 22 23:20:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:15:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 23:00:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:55:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:50:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:45:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:40:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:30:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:25:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:22:48 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-A4C1380D3B10-cache.txt
Oct 22 22:20:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:15:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:10:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-month.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:05:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 22:00:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:55:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:50:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:45:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:40:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-week.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:35:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:30:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:25:01 WimPi4-Sola goveebttemplogger[926]: Writing: /var/www/html/goveebttemplogger/gvh-A4C1380D3B10-day.svg With Title: H5177 (A4:C1:38:0D:3B:10)
Oct 22 21:21:48 WimPi4-Sola goveebttemplogger[926]: Writing: /var/cache/goveebttemplogger/gvh-A4C1380D3B10-cache.txt
```

```
wim@WimPi4-Sola:~ $ tail /var/log/goveebttemplogger/gvh-A4C1380D3B10-2024-10.txt
2024-10-23 07:05:10     -52.4287        28.7    0
2024-10-23 07:05:12     -52.4287        28.7    0
2024-10-23 07:05:14     -52.4287        28.7    0
2024-10-23 07:05:16     -52.4287        28.7    0
2024-10-23 07:05:20     -52.4287        28.7    0
2024-10-23 07:05:21     -52.4287        28.7    0
2024-10-23 07:05:23     -52.4287        28.7    0
2024-10-23 07:05:25     -52.4287        28.7    0
2024-10-23 07:05:34     -52.4287        28.7    0
2024-10-23 07:05:36     -52.4287        28.7    0
```

```
wim@WimPi4-Sola:~ $ grep --invert-match --fixed-strings '52.4287' /var/log/goveebttemplogger/gvh-A4C1380D3B10-2024-10.txt | tail --lines=50
2024-10-22 19:35:27     13.1579 57.9    0
2024-10-22 19:35:31     13.1579 57.9    0
2024-10-22 19:42:08     13.258  58      0
2024-10-22 19:42:11     13.258  58      0
2024-10-22 19:42:15     13.258  58      0
2024-10-22 19:42:23     13.258  58      0
2024-10-22 19:44:36     13.1581 58.1    0
2024-10-22 19:44:43     13.1581 58.1    0
2024-10-22 19:44:47     13.1581 58.1    0
2024-10-22 19:44:50     13.1581 58.1    0
2024-10-22 19:46:59     13.2581 58.1    0
2024-10-22 19:47:02     13.2581 58.1    0
2024-10-22 19:47:05     13.2581 58.1    0
2024-10-22 19:47:09     13.2581 58.1    0
2024-10-22 19:47:13     13.2582 58.2    0
2024-10-22 19:47:17     13.2582 58.2    0
2024-10-22 19:47:23     13.2582 58.2    0
2024-10-22 19:47:27     13.2582 58.2    0
2024-10-22 19:47:31     13.2582 58.2    0
2024-10-22 19:47:35     13.2582 58.2    0
2024-10-22 19:47:38     13.2582 58.2    0
2024-10-22 20:48:54     13.3592 59.2    0
2024-10-22 20:49:02     13.3592 59.2    0
2024-10-22 20:49:07     13.3592 59.2    0
2024-10-22 20:49:14     13.3593 59.3    0
2024-10-22 20:49:19     13.3593 59.3    0
2024-10-22 20:49:28     13.3593 59.3    0
2024-10-22 20:49:33     13.3593 59.3    0
2024-10-22 20:49:42     13.3594 59.4    0
2024-10-22 20:49:56     13.3593 59.3    0
2024-10-22 20:50:00     13.3593 59.3    0
2024-10-22 20:50:04     13.3594 59.4    0
2024-10-22 20:50:09     13.3593 59.3    0
2024-10-22 20:50:13     13.3593 59.3    0
2024-10-22 20:50:17     13.3593 59.3    0
2024-10-22 20:50:22     13.3593 59.3    0
2024-10-22 20:50:26     13.3593 59.3    0
2024-10-22 20:50:31     13.3593 59.3    0
2024-10-22 20:50:35     13.3594 59.4    0
2024-10-22 20:50:48     13.3593 59.3    0
2024-10-22 20:50:57     13.3593 59.3    0
2024-10-22 20:51:02     13.3593 59.3    0
2024-10-22 20:51:06     13.3594 59.4    0
2024-10-22 20:51:24     13.3594 59.4    0
2024-10-22 20:51:28     13.3594 59.4    0
2024-10-22 20:51:33     13.3594 59.4    0
2024-10-22 20:51:46     13.3594 59.4    0
2024-10-22 20:51:50     13.3594 59.4    0
2024-10-22 20:51:54     13.3594 59.4    0
2024-10-22 20:51:58     13.3594 59.4    0
```

**Comment by wcbonner** [comment]
https://github.com/wcbonner/GoveeBTTempLogger/commit/6bec69e13a69f167728d04891ed9386f632da16c should have fixed the error. I still don't understand the cause. I've added further repetition of setting the Bluetooth filter in more recent code.
